client/src/pages/Configurations/ConfigBuild/index.tsx

```diff
@@ -908,14 +908,18 @@ export function CustomCodeModal({
             type="text"
             value={form.code}
             onChange={(e) => {
-              setForm((prev) => ({ ...prev, code: e.target.value }));
+              const value = e.target.value.trimStart(); // trim leading space only while typing
+              setForm((prev) => ({ ...prev, code: value }));
               if (error) setError(''); // clear error on change
             }}
             onBlur={() => {
-              if (loincCodes.includes(form.code)) {
+              const trimmedCode = form.code.trim(); // full trim (leading + trailing)
```

Comment:
is there a reason why we needed to ad in this trim processing? wondering what the extra / discovered error state was